test/e2e/framework/kubelet_stats.go

```diff
@@ -367,6 +367,11 @@ func getOneTimeResourceUsageOnNode(
 	}
 
 	f := func(name string, newStats *stats.ContainerStats) *ContainerResourceUsage {
+		// TODO(gmarek): remove when #46198 is debugged.
+		if newStats == nil || newStats.CPU == nil {
```

Comment:
Also add `newStats.CPU.UsageNanoCores == nil` ? Because it might be nil. And I guess most probably a nil UsageNanoCores causes #46198